pkg/config/config.go

```diff
@@ -160,7 +160,8 @@ var defaultServerConfig = &ServerConfig{
 		NumWorkerPoolGoroutine: 16,
 		SortDir:                "/tmp/cdc_sort",
 	},
-	Security: &SecurityConfig{},
+	Security:            &SecurityConfig{},
+	PerTableMemoryQuota: 20 * 1024 * 1024, // 20MB
```

Comment:
Is `20MB` too small?

Reply:
The `20M` is for RawKV, not taking into consideration the data structures generated by the mounter.

Reply:
Could you add benchmark about the throughput of the flow controller with `20MB` as memory quota

Reply:
Done